venues/auai.org/UAI/2017/python/config.py

```diff
@@ -8,6 +8,8 @@
 PC = "auai.org/UAI/2017/Program_Committee"
 CONFERENCE = "auai.org/UAI/2017"
 SUBMISSION = "auai.org/UAI/2017/-/blind-submission"
+METADATA = "auai.org/UAI/2017/-/Paper_Metadata"
+ASSIGNMENT = 'auai.org/UAI/2017/-/Matching_Assignments'
```

Comment:
I feel like Matching_Assignments is a little redundant. Could we make it Reviewer_Assignments?
Also, this string should probably use double quotes like the others.